platforms/hyperledger-fabric/configuration/roles/upgrade/orderer-binary/tasks/create_orderer_value.yaml

```diff
@@ -0,0 +1,94 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+############################################################################################
+# This task creates the value file for the Orderers as per requirements mentioned in
+# network.yaml
+- name: Create orderers value file
+  include_role:
+    name: helm_component
+  vars:
+    name: "orderer"
+    org_name: "{{ item.name | lower }}"
+    component_name: "{{ orderer.name }}-{{ org_name }}"
+    orderer_image: "hyperledger/fabric-orderer:{{ network.version }}"
+    alpine_image: "{{ docker_url }}/alpine-utils:1.0"
+    type: "orderers"
+    consensus: "{{component_services.consensus}}"
+    genesis: "{{ lookup('file', '{{ build_path }}/channel-artifacts/genesis.block.base64') }}"
+
+#Git Status : check if there is any files modified in the directory
+- name: Git Status
```

Comment:
This is not required. Also, I see you registered the output of this task to `git_result` and in the very next task also, the registered variable is `git_result` which is useless